biscuit-auth/Cargo.toml

```diff
@@ -64,23 +63,6 @@ codspeed-bencher-compat = "2.6.0"
 #[build-dependencies]
 #prost-build = "0.10"
 
-[package.metadata.capi.library]
-# Used as the library name and defaults to the crate name. This might get
-# prefixed with `lib` depending on the target platform.
-name = "biscuit_auth"
-
-include = [
-  "Cargo.toml",
-  "cbindgen.toml",
-  "build.rs",
-  "examples/*.rs",
-  "LICENSE",
-  "README.md",
-  "src/*.rs",
-  "src/*/*.rs",
-  "tests/*.rs"
-]
```

Comment:
I didn't backported this, if someone has a clue on what's the purpose of `include` config